ingest-app/docs/api.md

```diff
@@ -687,7 +682,7 @@ Service concept can be created or updated by sending an HTTP PUT with the metada
 ```
 curl -i -XPUT \
   -H "Content-type: application/vnd.nasa.cmr.umm+json" \
-  -H "Echo-Token: XXXX" \
+  -H "Authorization: Bearer XXXX" \
   %CMR-ENDPOINT%/providers/PROV1/services/service123 \
   -d \
 "{\"Name\": \"AIRX3STD\",  \"Type\": \"OPeNDAP\",  \"Version\": \"1.9\",  \"Description\": \"AIRS Level-3 retrieval product created using AIRS IR, AMSU without HSB.\",  \"OnlineResource\": {    \"Linkage\": \"https://acdisc.gesdisc.eosdis.nasa.gov/opendap/Aqua_AIRS_Level3/AIRX3STD.006/\",    \"Name\": \"OPeNDAP Service for AIRS Level-3 retrieval products\",    \"Description\": \"OPeNDAP Service\"  },  \"ServiceOptions\": {\"SubsetType\": [\"Spatial\", \"Variable\"],    \"SupportedProjections\": [\"Geographic\"], \"SupportedFormats\": [\"netCDF-3\", \"netCDF-4\", \"Binary\", \"ASCII\"]}}"
```

Comment:
since your in the code, I think you should update the example json to the latest service (1.5) as outdated metadata examples may result in the generation of support emails which someone has to respond to. https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/browse/service/v1.5.0